cve_bin_tool/cli.py

```diff
@@ -279,13 +279,11 @@ def main(argv=None):
         "--epss-percentile",
         action="store",
         help="minimum epss percentile of CVE range between 0 to 100 to report (default: 0)",
```

Comment:
```suggestion
        help="minimum epss percentile of CVE range between 0 to 100 to report",
```
If we're removing the default to make the logic work better, we should also remove the default shown in help text.